frontend/src/components/layout/Tabs.vue

```diff
@@ -52,9 +52,9 @@ const headers = computed(() =>
     click: () => (activeTab.value = tab),
     classes: {
       'cursor-pointer': tab.label != activeTab.value.label,
-      'bg-teal-very-dark/40': tab.label != activeTab.value.label,
-      'text-teal-light': tab.label == activeTab.value.label,
-      'text-teal-light/40': tab.label != activeTab.value.label,
+      'bg-black/40': tab.label != activeTab.value.label,
+      'text-sea-green': tab.label == activeTab.value.label,
+      'text-sea-green/40': tab.label != activeTab.value.label,
     },
```

Comment:
Maybe we can simplify this by creating `inactiveTabClasses` & `activeTabClasses` and then inside the dom element assign the classes based on the condition `header.label == activeTab.label`.
The current approach makes all the header objects carry all this info with them which is unnecessary. 
Also these 4 equality checks are being evaluated each time per header element.

Reply:
Not really related to this PR, but yes, makes it much cleaner. I'll go for it.

Reply:
Shouldn't have requested changes, a simple comment would've been enough, but thanks for understanding the need for this!